### ISSUE TITLE
Error after install

### ISSUE THREAD

**Issue by tnoble** [litecommerce/core#15]
The moment install has completed, out of the box install, I get the following:

ERROR: "Includes\ErrorHandler::FATAL_ERROR" (code 2)

Call to a member function log() on a non-object

I have tried multiple installs but every time receive the same error.

**Comment by qimmik** [comment]
I'm not a designer nor do I work for LiteCommerce, but are you installing Drupal with LC3 or using the standalone LC3?

**Comment by tnoble** [comment]
I am using the package available on the "Downloads" page which includes Drupal and LC3.

When I try running the install package, regardless of whether I install "Typical", "Minimal" or "E-Commerce", whenever I enable (if it's not enabled by default in the instillation option) LC3, my website returns the above mentioned error.

**Comment by svowl** [comment]
Please provide a link from where you downloaded the file

**Comment by tnoble** [comment]
http://www.litecommerce.com/download.html

That is the page I downloaded it from with the following file link:

http://www.litecommerce.com/download/ecms

**Comment by svowl** [comment]
Ok. Could you please post a ticket to http://bt.litecommerce.com and attach log files from the directory drupal-dir/modules/lc_connector/litecommerce/varl/log (choose 'Support communications' category for the ticket)?

**Comment by tnoble** [comment]
I have managed to get the site to load correctly by:

1.Reuploading all files
2.Running "install.php" again
3.Refreshing every page that comes up
4.Ignoring errors telling me there are HTTP requests to log() that could not be resolved

I can supply a copy of the log files if you feel this may help for future releases?

As a side note, the site (although slightly limited by my hosting provider) is extremely slow in comparison to other Drupal 7 sites.

**Comment by qimmik** [comment]
So all is working now? That's great if it is! I have noticed that the site gets faster the longer it is up and running, probably cause it creates a cache for faster access. So, maybe in time it will load faster and faster. Again, I am not a dev or anything, just helping where I can cause I love this product!

**Comment by tnoble** [comment]
Yes :) I am finding the very same, before my page loading times were between 20 and 45 seconds now they average around 5 seconds (which is probably because my hosting provider is bottle necking my bandwidth). This is a very good product and I am extremely impressed with it. I tried implementing Ubercart in the past (before I found this) and had very limited success. Everything I wanted to do with that is included out of the box in this product with no effort required on my part what so ever. Big credit to all devs who have helped with the development and continual maintenance of this product.
